specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-05-01-preview/inference.json

```diff
@@ -0,0 +1,1249 @@
+          "$ref": "#/definitions/SkuScaleSettings"
+        },
+        "environmentConfigId": {
+          "type": "string",
+          "description": "Resource Id of the Environment Configuration artifact."
+        },
+        "codeAssetId": {
+          "type": "string",
+          "description": "Resource Id of the Code Assets i.e. startup code."
+        },
+        "vnet": {
+          "description": "Info of the customer's custom Virtual Network",
+          "properties": {
+            "id": {
+              "type": "string",
+              "description": "ResourceId of the Virtual Network."
+            },
+            "subnetName": {
+              "type": "string",
+              "description": "Name of the Subnet in the Virtual Network."
+            }
+          }
+        }
+      }
+    },
+    "SkuScaleSettings": {
+      "description": "Description of the Deployment scale options.",
+      "type": "object",
+      "required": [
+        "minimum",
+        "maximum"
+      ],
+      "properties": {
+        "minimum": {
+          "format": "int32",
+          "description": "Minimum number of instances for this SKU.",
+          "type": "integer",
+          "minimum": 1,
+          "exclusiveMinimum": true
+        },
+        "maximum": {
+          "format": "int32",
+          "description": "Maximum number of workers for this SKU.",
+          "type": "integer"
+        },
+        "default": {
+          "format": "int32",
+          "description": "Default number of workers for this SKU.",
+          "type": "integer"
+        },
+        "scaleType": {
+          "description": "Available scale configurations for this Deployment.",
+          "type": "string"
+        }
+      }
+    },
+    "SkuInfo": {
+      "description": "SKU discovery information.",
+      "type": "object",
+      "properties": {
+        "resourceType": {
+          "description": "Resource type that this SKU applies to.",
+          "type": "string"
+        },
+        "sku": {
+          "$ref": "#/definitions/Sku",
+          "description": "Name and tier of the SKU."
+        },
+        "capacity": {
+          "$ref": "#/definitions/SkuScaleSettings",
+          "description": "Min, max, and default scale values of the SKU."
+        }
+      }
+    },
+    "SkuInfoCollection": {
+      "description": "Collection of SKU information.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SkuInfo"
+          }
+        }
+      }
+    },
+    "InferenceSku": {
+      "description": "Description of a SKU for a scalable resource.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Name of the sku",
+          "type": "string"
+        },
+        "tier": {
```

Comment:
My understanding is that we need some value on tier for azmon autoscale right - in this case we should suggest just one for now - I suggest we start with only "Basic"... also lets bring that up in the internal review

Reply:
This is the response when doing GET deployment. This is required for AzMon in their contract. This is not part of the request for deployment creation.

Reply:
Wait this is also used in Update deployment but this was also in the contract for update.